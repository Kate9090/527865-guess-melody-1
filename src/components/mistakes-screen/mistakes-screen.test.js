import React from 'react';
import renderer from 'react-test-renderer';
import MistakesScreen from './mistakes-screen.jsx';

describe(`GameMistakes`, () => {
  it(`renders correctly with 1 mistakes`, () => {
    const tree = renderer.create(<MistakesScreen mistakes={1} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly without mistakes`, () => {
    const tree = renderer.create(<MistakesScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
