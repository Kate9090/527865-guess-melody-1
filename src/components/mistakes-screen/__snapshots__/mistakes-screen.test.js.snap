// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameMistakes renders correctly with 1 mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;

exports[`GameMistakes renders correctly without mistakes 1`] = `
<div
  className="game__mistakes"
/>
`;
